app/views/welcome/_header.html.erb

```diff
@@ -20,8 +19,17 @@
 </nav>
 
 <ul class="sidenav" id="mobile-demo">
-  <li><a href="sass.html">About us</a></li>
-  <li><a href="badges.html">FAQ</a></li>
-  <li><a href="collapsible.html">Contact</a></li>
-  <li><a href="mobile.html">Login</a></li>
+  <li> | <%= link_to t('header.about_us'), pages_about_path %></li>
+  <li><%= link_to t('header.faq'), pages_faq_path %></li>
+  <li><%= link_to t('header.contact'), pages_contact_path %></li>
+  <div class='session-container'>
+    <% if user_signed_in? %>
+      <li>Logged in as <strong><%= current_user.email %></strong>.</li>
```

Comment:
Please internationalize this text. You can use interpolation on your strings, you can have this on your locale file 
```
header:
  logged_user: 'Logged in as %{user_email}.'
```
and here you can change this by
```
<li><%= t('header.logged_user', user_email: current_user.email).html_safe %></li>
```

We're doing two things here
- Using variables on the internationalized strings.
- Using HTML tags directly from that text.